Makefile

```diff
@@ -36,11 +36,11 @@ bootstrap-with-docker: generate-version-file ## Build the image to run the app i
 
 .PHONY: run-flask
 run-flask: ## Run flask
-	. environment.sh && flask run -p 6011
+	. environment.sh && gunicorn -b 127.0.0.1:6011 application
```

Comment:
I think we should keep this command as was before. This command is only used by developers to run the API locally. 99.9% of the time, they only need to run flask, they don't need to run gunicorn infront of it (they would only need to run gunicorn infront if they were editing gunicorn configuration which is done extremely rarely).

Being able to just run flask locally as before will help keep it simple for local development for anyone who doesn't run it in docker.

Reply:
definitely want to keep this - `flask` runs in development mode and offers hot code reloading when source files are modified